src/client/index.js

```diff
@@ -75,7 +75,7 @@ const app = {
 const front = new Front("app", app, config);
 
 front.start(true);
-front.store.dispatch(setMessage({ message: "Welcome Dude !" }));
+front.store.dispatch(setMessage("Welcome Dude !"));
```

Comment:
this has changed recently I guess. can we update the content to "Welcome Human!"?